doctor/cores/text_processor.py

```
@@ -0,0 +1,462 @@
+"""
+هسته پردازش متن اپ Doctor
+Doctor App Text Processor Core
+"""
+
+import logging
+import re
+from typing import Dict, List, Any, Optional, Tuple
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)